docker-compose.yml

```
@@ -10,6 +10,8 @@ services:
       - PORT=80
       - VIRTUAL_HOST=reverifi.docker
       - REACT_APP_API_URL=http://reverifi-api.docker
+      - REACT_APP_FACEBOOK_APP_ID=348656156906042
+      - REACT_APP_GOOGLE_CLIENT_ID=277465120739-3k8i62n20e9fg3hv1rkcd32agkv0c4ak.apps.googleusercontent.com
```

```suggestion
      - REACT_APP_FACEBOOK_APP_ID
      - REACT_APP_GOOGLE_CLIENT_ID
      ```
      
      Please remove these secrets from the code and only keep them in your .env file.